react-native/react/native/pinentry-listeners.desktop.js

```diff
@@ -0,0 +1,36 @@
+export default {
+  'keybase.1.secretUi.getPassphrase': (payload, response) => {
+    console.log('Asked for passphrase')
+    /*
```

Comment:
You can type type annotate this! https://github.com/keybase/client/blob/master/protocol/js/flow-types.js#L2755